commit f40b7a9f34c554213d1002302a9c3f87b11ce1e4
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-03-18T13:23:49-04:00 (7 months ago)
    
    TOOLS-2101 jsstyle submodules too old for old perls
    Reviewed by: Cody Peter Mello <melloc@writev.io>
    Approved by: Cody Peter Mello <melloc@writev.io>

diff --git a/.gitmodules b/.gitmodules
index 14e5f4b..6998f69 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle.git
+	url = https://github.com/joyent/jsstyle.git
diff --git a/deps/jsstyle b/deps/jsstyle
index b94eb62..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit b94eb629ee524d21b563c8b4461caa2bdf1becc6
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
